src/core_plugins/kibana/common/tutorials/filebeat_instructions.js

```diff
@@ -181,6 +181,7 @@ export const createFilebeatInstructions = () => ({
         '  password: "<password>"',
         'setup.kibana:',
         '  host: "<kibana_url>"',
+        `  space.id: "${context ? context.spaceId : '<space_id>' }"`
```

Comment:
you should ensure that `context.spaceId` exists. In Kibana OSS without the spaces x-pack plugin, context will just be `{}`.

Reply:
Sure, thanks!